### DISCUSSION TITLE
[12] Dinder - Testplan

### DISCUSSION THREAD

**Discussion by JannisFuchs** [dhbw-ka-tinf22b5-dinder/Dinder#17] [Projektblog]
# Testplan
![TestPlan](https://github.com/dhbw-ka-tinf22b5-dinder/Dinder-SRS/blob/main/testplan.md)
# Further changes
## Frontend
- Refactor of state management with Redux
- Migration from React and Vite to Next.js
## Backend
- Integration of the Kotlin library for Supabase
- Upload of arbitrary files to a backend endpoint (evaluation)

**Comment by mausio** [comment]
expensive testing!

**Comment by NeeroxZ** [comment]
Krass, wie gründlich ihr die verschiedenen Aspekte eurer Teststrategie durchdacht habt. Die klare Gliederung und die umfassende Abdeckung von Unit-, UI- und Integrationstests zeigen ein starkes Engagement für die Qualität eurer Anwendung.

Besonders positiv finde ich eure Betonung auf Automatisierung durch CI/CD-Pipelines und die Integration mit SonarCloud.

Eure Risikominderungsstrategien sind gut durchdacht und zeigen, dass ihr bereits im Voraus mögliche Probleme antizipiert und Pläne zur Bewältigung habt.

Insgesamt finde ich euren Testplan sehr professionell und gut durchdacht. Es ist klar, dass ihr euch intensiv mit den verschiedenen Aspekten eurer Teststrategie auseinandergesetzt habt, und ich bin zuversichtlich, dass eure Anwendung von diesem gründlichen Ansatz profitieren wird.

-IncidArch
